advocacy_docs/edb-postgres-ai/analytics/reference/queries.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Queries
+description: Supported queries in Lakehouse and best practices when composing them
+---
+
+In general, **READ ONLY** queries are supported. You cannot write directly to
+object storage. This includes all Postgres built-in functions, statements
+and types. It also includes any of those provided by EPAS or PGE, depending on
+which distribution you choose to deploy.
+
+In general, you cannot insert, update, delete or otherwise modify data. You
+cannot `CREATE TABLE`. You must load data into the bucket out-of-band, either
```

Comment:
Confusing, because now you can `CREATE TABLE`

More precisely, "you cannot `CREATE TABLE` to create a new Delta Table in a storage location. But you can use `CREATE TABLE` to create a normal Postgres table on the node (keeping in mind the node is ephemeral). And you can create "external" tables that are Postgres tables that point to Delta Tables in object storage (link to `external-tables doc`)"